### PR TITLE
Adding voice types for africastalking

### PR DESCRIPTION
**Pull request by james-kaguru**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- Updating types not already added

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/africastalking/index.d.ts
1. types/africastalking/africastalking-tests.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against b4ac53dd7ad6eac7cd53f0842c22daa7c97d2578
</p>

**Comment by james-kaguru** [comment]
Hello @jakebailey could you check this pr out

File: types/africastalking/index.d.ts [hunk 1]
```diff
@@ -38,29 +39,74 @@ interface SMS {
...
-export = africastalking;
+declare function africastalking(options: {
+    username: string;
+    apiKey: string;
+}): AfricasTalking;
 
-declare function africastalking(
-    options: { username: string; apiKey: string },
-): AfricasTalking;
+declare namespace africastalking {}
```
Comment (jakebailey) [inline types/africastalking/index.d.ts:111]: Why is this here? It's empty and couldn't do anything.

Comment (james-kaguru) [reply types/africastalking/index.d.ts:111]: In the previous method I found that I could get the types to work in the example file but this one does

Comment (gabritto) [reply types/africastalking/index.d.ts:111]: Did you mean to declare the interfaces inside the "africastalking" namespace, so that they are also exported and can be imported and used elsewhere?

**Review by gabritto — CHANGES_REQUESTED** [review]
Looks good except for the empty namespace, I left a question about that.

File: types/africastalking/index.d.ts (hunk 1 above)
Comment (james-kaguru) [reply types/africastalking/index.d.ts:111]: @gabritto  Its so that I can be able to get the types in the africastalking tests file..for some reason the earlier method didnt allow me to access types in the test folder but with that extra change they were now visible.

Comment (gabritto) [reply types/africastalking/index.d.ts:111]: Ok, I still think the interfaces declared here should be *inside* the namespace. Can you make that change?
